### PR TITLE
[release-3.7] Updating process for doing a rolling and full cluster upgrades

### PR DESCRIPTION
**Pull request by ewolinetz**
Manual cherrypick of https://github.com/openshift/openshift-ansible/pull/8415

### CODE REVIEW

File: roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml [hunk 1]
```diff
@@ -1,120 +1,87 @@
 ---
 ## get all pods for the cluster
 - command: >
+<<<<<<< HEAD
```
Comment (richm) [inline roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml]: git conflict markers

Comment (ewolinetz) [reply roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml]: I blame my file editor on that one...

**Pull request merged**